### ISSUE TITLE
Import/Export .md and .txt files to/from the database

### ISSUE THREAD

**Issue by KhaaL** [nuttyartist/notes#316]
As a enduser, in order to migrate to Notes I'd need to migrate my already made text-files. The only alternative is to copy-paste them, which would be very time consuming.

**Comment by nuttyartist** [comment]
I agree that's a good suggestion. Thanks!

**Comment by KhaaL** [comment]
Let me know if you'd like to have a more fleshed out user-story, I wrote the issue quickly just to get it off my head 😃

**Comment by nuttyartist** [comment]
I think it's a pretty understandable request. If you think there's something you should point out, please do.

**Comment by nuttyartist** [comment]
@batsatt You're right. We will ship this on our next (minor) release. See: https://github.com/nuttyartist/notes/projects/8#card-85711502

**Comment by badrow** [comment]
Very important feature,I need it too ! I have 1700+ notes in txt format, in 10 folders, and I don't know how to import them in this app. 
Thank you for your job, very interesting project !

**Comment by passionate2023** [comment]
Any update on this option?

**Comment by passionate2023** [comment]
Any chance of implementing this?

**Comment by nuttyartist** [comment]
I hope we'll manage to make it happen when releasing v2.3.0, fingers crossed.

**Comment by passionate2023** [comment]
Any update on this?

**Comment by nuttyartist** [comment]
Nothing new yet. No need to keep asking here, please. I'll keep you posted.

**Comment by passionate2023** [comment]
i meant when the 2.3.0 is expected - next year or this year? and not update on the about feature

**Comment by nuttyartist** [comment]
I'm not sure. I estimate it will take around 4 months. It's not going to be 2.3.0 but a major new version.

**Comment by passionate2023** [comment]
Is this feature is available now @nuttyartist ?

**Comment by nuttyartist** [comment]
Hey, not yet. Please ask again in 2 months from now.

**Comment by nuttyartist** [comment]
@passionate2023 I'm happy to finally say I implemented importing .txt/.md files in Plume, and it will see its way into Notes as well. In a few days I should implement exporting to .txt/.md as well.

**Comment by passionate2023** [comment]
Thank you so much @nuttyartist

**Comment by zjeffer** [comment]
Closing as completed.
